spec/components/search_spec.rb

```diff
@@ -792,6 +792,21 @@ def search
       expect(search.posts.length).to eq(2)
     end
 
+    it 'can use tag as a search context' do
+      tag = Fabricate(:tag, name: 'important-stuff')
+
+      topic = Fabricate(:topic)
+      topic_no_tag = Fabricate(:topic)
+      Fabricate(:topic_tag, tag: tag, topic: topic)
+
+      post = Fabricate(:post, topic: topic, user: topic.user, raw: 'This is my hello')
+      _another_post = Fabricate(:post, topic: topic_no_tag, user: topic.user)
```

Comment:
In this case it's a bit cleaner to not assign the variable rather than use the underscore to indicate it's not used.